src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritManagement.java

```diff
@@ -138,7 +142,9 @@ public String getDisplayName() {
          * @return the list of descriptors containing GerritServer's descriptor.
          */
         public static DescriptorExtensionList<GerritServer, GerritServer.DescriptorImpl> serverDescriptorList() {
-            return Jenkins.getInstance()
+            Jenkins jenkins = Jenkins.getInstance();
+            assert jenkins != null;
```

Comment:
Note that it _is_ valid for `Jenkins.getInstance` to be null during shutdown, so in general you need to think about whether your code might be running in this mode and behave gracefully if so.

There is no good solution for `DescriptorExtensionList`. For `ExtensionList` you can use the new `lookup` method to behave smoothly during shutdown.